### PR TITLE
Adaptive case support

### PR DESCRIPTION
**Pull request by ashinagawa**
This commit will add adaptive support back.

### CODE REVIEW

File: Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt [hunk 1]
```diff
@@ -209,6 +214,25 @@ class VungleBannerAdapter(adapter: VungleAdapter) :
 
             else -> null
         }
+
+        try {
+            if (bannerSize.isAdaptive && vungleAdSize != null) {
+                val context = ContextProvider.getInstance().applicationContext
+                val adaptiveSize = VungleAdSize.getAdSizeWithWidth(context,
```
Comment (ManojBudumuru-vungle) [inline Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt]: we r not suppose to use this. This api has some inconsistencies.
we should use
VungleAdSize.getAdSizeWithWidthAndHeight(width: bannerSize.containerParams.width, height: 0);

File: Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt [hunk 2]
```diff
@@ -218,8 +218,9 @@ class VungleBannerAdapter(adapter: VungleAdapter) :
         try {
             if (bannerSize.isAdaptive && vungleAdSize != null) {
                 val context = ContextProvider.getInstance().applicationContext
```
Comment (ManojBudumuru-vungle) [inline Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt]: u dont need this anymore i think. may be remove this line.

File: Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt [hunk 3]
```diff
@@ -218,8 +218,9 @@ class VungleBannerAdapter(adapter: VungleAdapter) :
         try {
             if (bannerSize.isAdaptive && vungleAdSize != null) {
```
Comment (ManojBudumuru-vungle) [inline Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt:219]: one more qn should we have to check if the vungleAdSize != null here? just to confirm.

Comment (ashinagawa) [reply Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt:219]: Yes. We need it. After removing it, I got compile error.

**Review by HuFangshuai — APPROVED** [review]

**Pull request merged**